maudtools/generating_priors.py

```diff
@@ -0,0 +1,43 @@
+"""Functions for generating a prior file format for kinetic parameters""" 
+
+import pandas as pd
+from maud.data_model.stan_variable_set import StanVariableSet
+
+
+def get_kinetic_variable_set_dataframe(
+    svs: StanVariableSet
+):
+    prior_dict_set = {}
+    for par_id, sv in svs.__dict__.items():
+        if (par_id != "__initialised__") and (sv is not None):
+            prior_dict_set[par_id] = pd.DataFrame.from_dict({
+                sv.id_components[0][i].value:
+                sv.split_ids[i]
+                for i in range(len(sv.id_components[0]))
```

Comment:
just an idea - the `i` here this can be avoided using `zip`:

```python
id_component.value: split_id
for id_component, split_id in zip(sv.id_components[0], sv.split_ids) 
```